app/packages/spaces/src/hooks.ts

```diff
@@ -100,6 +104,17 @@ export function usePanel(name: SpaceNodeType) {
   return panels.find((panel) => panel.name === name);
 }
 
+export function useReactivePanel(name: SpaceNodeType) {
```

Comment:
Add comment explaining why this is needed because the `count` variable here is going to throw us off int he future.